### DISCUSSION TITLE
chrome inspect and chrome developer tools blocked/disabled.

### DISCUSSION THREAD

**Discussion by catfoolyou** [3kh0/ext-remover#867] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Those assholes at our school district (FCPS) have found a way to disable Inspect and Chrome developer tools. Any way to get them unblocked?

Btw they are slowly moving to chromebooks, only a few lucky guys (myself included) still use Windows laptops

**Comment by Mazedotexe** [comment]
its crazy that you had devtools until now. i never had it allowed, and pretty much nobody but you and the people at your school have had that allowed. if you just want to run javascript, look into bookmarklets. if you want devtools, use the Eruda bookmarklet. if bookmarklets are disabled, try to find an extension you can run javascript with (shortkeys, ublock origin) and use Eruda.

**Reply by AshtonDavies** [reply]
Oh no, many other people do have it allowed for some reason.

**Reply by Mazedotexe** [reply]
wtf i refuse to believe im just unlucky lol

**Reply by AshtonDavies** [reply]
Fair enough

**Reply by Aka-but-nice** [reply]
Yeah, I'm not surprised, I used to be able to install any play store app I wanted. We all know how that ended, right?

**Reply by Mazedotexe** [reply]
flashbacks to my sysadmin saying "COOL it with the goguardian stuff, [name]. i know it's fun and exciting to you, but you can get in serious trouble for this."


...ok thanks for that nugget of information sysadmin 👍 👍

**Reply by AshtonDavies** [reply]
When they introduced the new Google account system, I added my personal Google account and it unblocked the Play Store to install any apps and stayed even after my district blocked multiple accounts. But, they completely disabled the Play Store later on because of constant discoveries of webview tricks being done by students and that's how it ended for me back then.

**Reply by Aka-but-nice** [reply]
:skull:? my school never went THAT far for those discoveries, didn't your school require an app or two for education purposes (even if they weren't used at all)?

**Reply by AshtonDavies** [reply]
I was surprised too, but apparently everything they really needed was in the Chrome Web Store. Yeah, sure we had to give up some apps like Libby which is now in the browser.

**Reply by Aka-but-nice** [reply]
Anyways, it's practically useless since both play store apps and chrome apps have the webview permission. So whether they block the play store or the web store, there won't be much differences (besides the fact how the appchrome apps parses browser polices).

**Reply by AshtonDavies** [reply]
I know, but my district was dumb and didn't know how to solve that simple problem, lol (they seemingly didn't know how to check for webview bypasses in apps). Especially after getting my own LoMoH exploit patched thanks to my darn district, they were still dumb as heck to not realize it was one of their students behind it all and simply reported and blocked it.

**Reply by catfoolyou** [reply]
Well FCPS has some smart-ass mfers on the job because bookmarklets are blocked, non-administrated (non school) accounts cannot be added and only "approved" extensions can be installed. Thinking of reinstalling Windows on my machine.

**Reply by Aka-but-nice** [reply]
That's... average policy? I'm not seeing any abnormalities within your gavengiven privileges, those are more common than you think 🙄 .

**Comment by tuta-amb** [comment]
Looks like pretty standard policies. If you're on a Windows computer, there are like a trillion exploits you can do, Windows is pretty hard to lock down. If you're on ChromeOS, what version are you on?

**Comment by S-PScripts** [comment]
closed unless you still need an answer

**Reply by S-PScripts** [reply]
if you do join our discord
